src/app/themes.tsx

```diff
@@ -184,7 +178,7 @@ const StyleManager: React.FC<Props> = ({ children, theme = lightTheme }) => {
             },
         },
         typography: {
-            fontFamily: "Inter",
+            fontFamily: "Inter-Regular, sans-serif",
```

Comment:
Not 100% sure of this by any means, but not sure sans-serif actually refers to a font, think it instead refers to a category of fonts, in which case we could do with a specific backup font (e.g. Arial or Helvetica)

Reply:
Okie added the two mentioned